recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -30,6 +30,9 @@ GPIO_IR_TX ?= "17"
 
 CAN_OSCILLATOR ?= "16000000"
 
+WM8960="${@bb.utils.contains("MACHINE_FEATURES", "wm8960", "1", "0", d)}"
+WM8960DTBO="wm8960-soundcard"
```

Comment:
This feels like an undeeded extra variable.

Reply:
Is `WM8960DTBO` any different from `VC4DTBO`? I shall eliminate it.

Reply:
Yes, it is because that is using a weak assignment leaving the user the ability to overwrite. I don't see that as relevant in this case. Or is it?

Reply:
Makes sense now. There's no need for wm8960-soundcard dtbo to be overridden.